File: python/03-repeticao/main050.py
'''Desenvolva um programa que faça o sorteio de 20 números entre 0 e 10 e 
mostre na tela: 
a) Quais foram os números sorteados 
b) Quantos números estão acima de 5 
c) Quantos números são divisíveis por 3 

Data 17/10/2025 -> 25ª Dia
'''

import random

sorteio = []
acima_5 = 0
divi_3 = 0

for _ in range(20):
    numero = random.randint(1,10)
    sorteio.append(numero)
    if numero > 5:
        acima_5 += 1
    elif numero % 3 == 0:
        divi_3 += 1
print(f'a) Números sorteados: {sorteio}')
print(f'b) Números acima de 5: {acima_5}')
print(f'c) Números divisíveis por 3: {divi_3}')

File: c/03-repeticao/main050.c
/*Desenvolva um programa que faça o sorteio de 20 números entre 0 e 10 e 
mostre na tela: 
a) Quais foram os números sorteados 
b) Quantos números estão acima de 5 
c) Quantos números são divisíveis por 3 

Data 17/10/2025 -> 25ª Dia
*/

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {

    srand(time(NULL));

    int numero[20];
    int acima_5 = 0;
    int divi_3 = 0;

    printf("a) Numero sorteados: ");
    for(int i = 0; i < 20; i++) {

        numero[i] = rand() % 11;
        printf("%d ",numero[i]);

        if(numero[i] > 5) {
            acima_5 += 1;
        }
        else if(numero[i] % 3 == 0) {
            divi_3 += 1;
        }
    }
    printf("\nb) Acima de 5: %d\n",acima_5);
    printf("c) Divisiveis por 3: %d\n",divi_3);
    return 0;

}